README.md

```diff
@@ -1 +1,63 @@
-# Leaderboard-Project
\ No newline at end of file
+# LeaderBoard
+
```

Comment:
-[ ] Great job so far. The wireframe looks ok, styling is on point. please kindly add a good description to the readme about the app and how it functions.  It should have instructions about where and how to generate any necessary keys and credentials, how to set, store, and generate them

Reply:
Thank you so much for the review, I will implement the changes advised.